Up2dateService/Up2dateClient/Client.cs

```diff
@@ -151,31 +149,14 @@ private void OnDeploymentAction(IntPtr artifact, DeploymentInfo info, out Client
             }      
             
             var filePath = Path.Combine(getDownloadLocation(), info.artifactFileName);
-            if (settingsManager.CheckSignature && !certificateManager.IsSigned(filePath))
-            {
-                File.Delete(filePath);
-                result.Message = "File not signed. File deleted";
-                result.Success = false;
-                WriteLogEntry(result.Message, info);
-                return;
-            }
-
-            if(settingsManager.InstallAppFromSelectedIssuer && !certificateManager.IsSignedByIssuer(filePath))
-            {
-                result.Message = "File not signed by selected issuer. File deleted";
-                result.Success = false;
-                File.Delete(filePath);
-                WriteLogEntry(result.Message, info);
-                return;
-            }
 
             if (IsSupported(info))
```

Comment:
Now this check seems redundant (see line 116)

Reply:
Fixed